app/controllers/records_controller.rb

```diff
@@ -63,7 +63,7 @@ def download_mpl
         send_data file.read, type: 'application/zip', disposition: 'attachment', filename: "bundle_#{bundle.version}_mpl.zip"
       end
     else
-      render body: nil, :status => :bad_request
+      render :body => nil, :status => :bad_request
```

Comment:
In #1087 we're changing away from hashrocket syntax. Can we do the same here, or do we need to keep it?